src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -416,6 +416,19 @@ export const ReviewRoute: FC<ReviewProps> = props => {
         })
         break
       }
+      case "credit_card_deactivated": {
+        const title = "Unable to process card"
+        const message =
+          "This card is inactive or no longer available. Please confirm with your card issuer if this card is active, try another payment method, or contact orders@artsy.net."
```

Comment:
**comment (non-blocking):** I started building a [utility](https://github.com/artsy/force/blob/af426cba9b2ace3abe27b2d1e1a3c5d8bafb6fca/src/Apps/Order/Utils/getErrorDialogCopy.tsx#L61) to replace instances of "orders@artsy.net" with `mailto` links in the error messages that we show users to encourage them to actually contact us. Let's see if we can use (or improve) that utility going forward.